.jenkins/pipelines/scheduled-tests.Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+/* A Jenkins pipeline that will handle code coverage and nightly tests
+*  These are the original pipelines:
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-code-coverage-nightly.yml
```

Comment:
Nit: Since your intention is to replace the nightly pipeline too, please also mention the original nightly pipeline here